docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -0,0 +1,284 @@
+[[size-your-shards]]
+== How to size your shards
+++++
+<titleabbrev>Size your shards</titleabbrev>
+++++
+
+To protect against hardware failure and increase capacity, {es} stores copies of
+an index’s data across multiple shards on multiple nodes. The number and size of
+these shards can have a significant impact on your cluster's health. One common
+problem is _oversharding_, a situation in which a cluster with a large number of
+shards becomes unstable.
+
+[discrete]
+[[create-a-sharding-strategy]]
+=== Create a sharding strategy
+
+The best way to prevent oversharding and other shard-related issues
+is to create a sharding strategy. A sharding strategy helps you determine and
+maintain the optimal number of shards for your cluster while limiting the size
+of those shards.
+
+Unfortunately, there is no one-size-fits-all sharding strategy. A strategy that
+works in one environment may not scale in another. A good sharding strategy must
+account for your infrastructure, use case, and performance expectations.
+
+The best way to create a sharding strategy is to benchmark your production data
+on production hardware using the same queries and indexing loads you'd see in
+production. For our recommended methodology, watch the
+https://www.elastic.co/elasticon/conf/2016/sf/quantitative-cluster-sizing[quantitative
+cluster sizing video]. As you test different shard configurations, use {kib}'s
+{kibana-ref}/elasticsearch-metrics.html[{es} monitoring tools] to track your
+cluster's stability and performance.
+
+The following sections provide some reminders and guidelines you should consider
+when designing your sharding strategy. If your cluster has shard-related
+problems, see <<fix-an-oversharded-cluster>>.
+
+[discrete]
+[[shard-size-reminders]]
+=== Things to remember
```

Comment:
"Things to remember" seems a bit vague. Maybe "Sizing considerations" and then use "things" in the intro sentence?